ui-components/src/global/text.scss

```diff
@@ -140,42 +115,85 @@ h1.title {
   @apply block;
 }
 
-.card-header {
-  font-family: var(--card-header-font-family, var(--bloom-font-alt-sans));
-  font-weight: var(--card-header-font-weight, 600);
-  color: var(--card-header-color, var(--bloom-color-blue-700));
-  font-size: var(--card-header-font-size, var(--bloom-font-size-3xl));
-  margin-block: var(--card-header-margin-block, 0 var(--bloom-s3));
-  line-height: var(--card-header-line-height, var(--bloom-line-height-tight));
-}
-
-.card-subheader {
-  font-family: var(--card-subheader-font-family, var(--bloom-font-alt-sans));
-  font-weight: var(--card-subheader-font-weight, normal);
-  color: var(--card-subheader-color, var(--bloom-color-black));
-  font-size: var(--card-subheader-font-size, var(--bloom-font-size-base));
-  margin-block: var(--card-subheader-margin-block, 0 var(--bloom-s3));
-}
-
-.table-header {
-  font-family: var(--table-header-font-family, var(--bloom-font-alt-sans));
-  font-weight: var(--table-header-font-weight, 600);
-  color: var(--table-header-color, var(--bloom-color-gray-800));
-  font-size: var(--table-header-font-size, var(--bloom-font-size-sm));
-  margin-block: var(--table-header-margin-block, 0 var(--bloom-s1));
-}
-
-.table-subheader {
-  font-family: var(--table-subheader-font-family, var(--bloom-font-alt-sans));
-  font-weight: var(--table-subheader-font-weight, normal);
-  color: var(--table-subheader-color, var(--bloom-color-gray-750));
-  font-size: var(--table-subheader-font-size, var(--bloom-font-size-sm));
-  margin-block: var(--table-header-margin-block, 0 var(--bloom-s3));
-}
-
-.category-header {
-  font-family: var(--category-header-font-family, var(--bloom-font-alt-sans));
-  font-weight: var(--category-header-font-weight, 600);
-  color: var(--category-header-color, var(--bloom-color-gray-700));
-  font-size: var(--category-header-font-size, var(--bloom-font-size-sm));
+:root {
+  --bloom-text-color-light: var(--bloom-color-gray-700);
+  --bloom-text-color: var(--bloom-color-gray-750);
+  --bloom-text-color-dark: var(--bloom-color-gray-800);
+  --bloom-text-color-darker: var(--bloom-color-gray-850);
+  --bloom-text-color-darkest: var(--bloom-color-black);
+}
+
+.text__large-primary {
+  font-family: var(--text-large-primary-font-family, var(--bloom-font-alt-sans));
+  font-weight: var(--text-large-primary-font-weight, 600);
+  color: var(--text-large-primary--color, var(--bloom-color-primary-dark));
+  font-size: var(--text-large-primary--font-size, var(--bloom-font-size-3xl));
```

Comment:
For this and line-height, looks like maybe there's an extra hyphen `-` in the middle of the var name?